packages/commons/src/main/util/Runtime.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Wire
+ * Copyright (C) 2018 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import * as platform from 'platform';
+import {BROWSER, WEBAPP_SUPPORTED_BROWSERS} from '../config/CommonConfig';
+
+class Runtime {
+  private static getPlatform(): Platform {
+    return platform;
```

Comment:
Return an empty object if instantiating platform failed otherwise all the later safeguard can be obsolete.

Reply:
Let's add a test for that case